### PR TITLE
Fixed the list scrolling 

### PR DESCRIPTION
**Pull request by aasthashah999**
### Overview 
The list no longer scrolls weirdly past the bottom sheet due to adding more layouts and fixing some attributes 

### Changes Made
Added more layouts and changed attributes

### CODE REVIEW

File: app/src/main/java/com/example/flick/NetworkUtils.kt [hunk 1]
```diff
@@ -136,4 +126,45 @@ class NetworkUtils {
...
+        val response = request?.let { client.newCall(it).execute() }
+        val body = response?.body
+        val stringBody = body?.string()
+        Log.d("JSON", stringBody)
+        val moshi = Moshi.Builder().build()
+        val adapater: JsonAdapter<SingleListAdapter.DataSingleMediaList> = moshi.adapter(
+            SingleListAdapter.DataSingleMediaList::class.java
+        )
+        return adapater.fromJson(stringBody)?.data
+    }
```
Comment (cindyh25) [inline app/src/main/java/com/example/flick/NetworkUtils.kt:169]: did you mean to just put this in the other PR 😛

File: app/src/main/res/layout/profile_list_bottom_sheet.xml [hunk 2]
```diff
@@ -3,42 +3,53 @@
...
         app:srcCompat="@drawable/add_list_button_final" />
 
+    <androidx.core.widget.NestedScrollView
+        android:layout_width="wrap_content"
+        android:layout_height="wrap_content"
+        app:layout_constraintStart_toStartOf="parent"
+        app:layout_constraintTop_toTopOf="parent"
+        android:overScrollMode="ifContentScrolls"
+        android:layout_marginTop="20dp"
+        >
```
Comment (cindyh25) [inline app/src/main/res/layout/profile_list_bottom_sheet.xml:32]: ooooh cool!!

**Review by cindyh25 — APPROVED** [review]
YAY looks good! 💯 💯

File: app/src/main/java/com/example/flick/NetworkUtils.kt (hunk 1 above)
Comment (aasthashah999) [reply app/src/main/java/com/example/flick/NetworkUtils.kt:169]: I think so, maybe I changed the spacing or something!

File: app/src/main/res/layout/profile_list_bottom_sheet.xml (hunk 2 above)
Comment (aasthashah999) [reply app/src/main/res/layout/profile_list_bottom_sheet.xml:32]: Thanks! I think the "ifContentScrolls" fixed it!

**Pull request merged**